src/game/camera.c

```diff
@@ -3460,7 +3460,7 @@ void evaluate_cubic_spline(f32 u, Vec3f Q, Vec3f spline1, Vec3f spline2, Vec3f s
     register f32 su = sqr(u);
     register f32 hcu = (su * u) / 2.0f;
```

Comment:
May as well remove these register keywords too

Reply:
i suppose i could, mostly just wanted to keep it to math_util